api/models/language_and_level.py

```diff
@@ -1,10 +1,14 @@
 from django.db import models
 
-from api.models.auxil.constants import LANGUAGE_AND_LEVEL_ID_TO_INDEX, LanguageLevelId
+from api.models.auxil.constants import LanguageLevelId
 
 # Sometimes the languages and levels are needed separately, sometimes in combinations.
 # This means that we cannot create a "choices" field for language or level.
 
+"""Dictionary with names of levels ("A0", "A1" etc.) as values 
+and their position in the enumeration as keys."""
+LANGUAGE_LEVEL_ID_TO_INDEX = {level: index for index, level in enumerate(sorted(LanguageLevelId))}
```

Comment:
nit: you have to put docstring below the constant for it to be picked up by IDE

Reply:
TIL :)